HomeTask_7

GaratovMD opened the pull request:
https://www.codewars.com/users/GaratovMD

GaratovMD committed:
HomeTask_7

https://www.codewars.com/users/GaratovMD

illarionovEA reviewed (changes requested):
темы должны интерпретироваться лучше, нужно подбирать параметры
получившиеся темы нужно соотнести с исходными темами из датасета

GaratovMD committed:
Delete HomeTask_7.ipynb

GaratovMD committed:
HomeTask_7

Исправил. Теперь лучше распределение по темам. Также явно разделил на темы
https://www.codewars.com/users/GaratovMD
